cmd/util/cmd/run-script/cmd.go

```diff
@@ -532,7 +532,32 @@ func (*api) SubscribeBlockDigestsFromLatest(
 	return nil
 }
 
-func (*api) SubscribeTransactionStatuses(
+func (a *api) SubscribeTransactionStatusesFromStartBlockID(
+	_ context.Context,
+	_ flow.Identifier,
+	_ flow.Identifier,
+	_ entities.EventEncodingVersion,
+) subscription.Subscription {
+	return nil
```

Comment:
instead of returning nil, how about returning a failing subscription with the error "not implemented"